nomanoma121/server/index.js

```diff
@@ -12,10 +14,30 @@ const todoList = [
   { id: 3, title: "学校の課題を提出する", completed: true },
 ];
 
+const schema = object({
```

Comment:
```suggestion
const Schema = object({
```

Valibot でスキーマつくるときは大文字から始めたほうがよさそう